src/pages/media.tsx

```diff
@@ -39,50 +42,79 @@ const pressReleases: Resource[] = [
-    date: "6 Aug 2020",
+    title: "Singapore, Chile and New Zealand Sign Digital Economy Partnership Agreement Electronically",
+    date: "12 Jun 2020",
     url:
-      "https://www.imda.gov.sg/news-and-events/Media-Room/Media-Releases/2020/Signing-of-Singapore-Australia-Digital-Economy-Agreement",
+      "https://www.imda.gov.sg/news-and-events/Media-Room/Media-Releases/2020/Singapore-Chile-and-New-Zealand-Sign-Digital-Economy-Partnership-Agreement-Electronically",
   },
   {
-    title: "SWIFT and Singapore's IMDA Join Forces to Drive Global Trade Digitalisation",
-    date: "5 Oct 2020",
+    title:
+      "Remarks by Mr S Iswaran, Minister for Communications and Information, at the International Chamber of Commerce 'Taking Trade Digital' Forum in Davos on 22 Jan 2020",
+    date: "22 Jan 2020",
     url:
-      "https://www.imda.gov.sg/news-and-events/Media-Room/Media-Releases/2020/SWIFT-and-Singapores-IMDA-Join-Forces-to-Drive-Global-Trade-Digitalisation",
+      "https://www.mci.gov.sg/pressroom/news-and-stories/pressroom/2020/1/Remarks%20by%20Minister%20S%20Iswaran%20at%20the%20International%20Chamber%20of%20Commerce#.X5gGtAayHwc.gmail",
+  },
+  {
+    title:
+      "World's Largest Business Organisation Joins Singapore Government and Major Industry Partners to Drive Global Digitalisation",
+    date: "22 Jan 2020",
+    url:
+      "https://www.imda.gov.sg/news-and-events/Media-Room/Media-Releases/2020/Worlds-Largest-Business-Organisation-Joins-Singapore-Government-and-Major-Industry-Partners-to-Drive-Global-Digitalisation",
+  },
+  {
+    title:
+      "Launch of TradeTrust announced by Mr S Iswaran, Minister for Communications and Information at the Committee of Supply Debate on 4 March 2019",
+    date: "4 Mar 2019",
+    url:
+      "https://www.mci.gov.sg/pressroom/news-and-stories/pressroom/2019/3/securing-our-future-in-a-digital-age#.X5gFH3jrCCc.gmail",
   },
 ];
 
-const eventsMedia = [
+const eventsMedia: ResourceEventProps[] = [
   {
-    title: "The Journey to Paperless Trade: Industry Initiatives for Interoperability",
+    title: "Digitalise Your Supply Chains to save cost, improve efficiency & Overall Profitability with Trames",
     description:
-      "This event by BAFT and R3 explored different blockchain-based digital trade solutions and how global trade participants can use these solutions towards challenges in the global trade ecosystem.",
+      "Designed for businesses of all sizes, learn how you can digitise your supply chains to save cost, improve efficiency and overall profitability by using the Trames platform.",
     resource: {
-      dateTime: "2 July | 9-10pm SGT",
-      watchLink: "https://www.youtube.com/watch?v=iRmaQV9HERw",
-      eventLink: "https://www.baft.org/baft-education/e-learning/past-webinars",
-      eventSlides:
-        "https://baft.org/docs/default-source/past-webinars/webinar-the-journey-to-paperless-trade.pdf?sfvrsn=6e9a20e1_2",
+      dateTime: "8 Apr 2021, 3-4pm (GMT+8)",
```

Comment:
here is even worse.
dateTime, then some are just dates.

Reply:
separated time from date since not all events has time, will figure out on timezone later in the cms pr